### ISSUE TITLE
log messages don't show in console output when ansi logging is enabled

### ISSUE THREAD

**Issue by phue** [nextflow-io/nextflow#1129]
### Expected behavior and actual behavior

Expected behaviour: `log.info "foo"` should print to STDOUT.
Actual behaviour: `log.info "foo"` only prints to .nextflow.log _unless_ `NXF_ANSI_LOG=false`

### Steps to reproduce the problem

`NXF_ANSI_LOG=false nextflow run nf-core/hlatyping -r dev -profile test` -> prints nfcoreHeader() to console
`nextflow run nf-core/hlatyping -r dev -profile test` -> doesn't print nfcoreHeader() to console

### Environment 

* Nextflow version: [19.04.0] 
* Java version: [1.8.0_202]
* Operating system: [Linux]

**Comment by pditommaso** [comment]
Not able to replicate this. Do you mean a script with just the line  `log.info "foo"` does not print it ?

**Comment by phue** [comment]
That example was maybe too minimal, sorry. It seems to me that with the ansi logging, not all log.info statements get printed to console. They do get printed to .nextflow log however.

For example the following script:

```
#!/usr/bin/env nextflow

params.monochrome_logs = false

def helpMessage() {
    log.info nfcoreHeader()
    log.info "foo"
}

def nfcoreHeader(){
    // Log colors ANSI codes
    c_reset = params.monochrome_logs ? '' : "\033[0m";
    c_dim = params.monochrome_logs ? '' : "\033[2m";
    c_black = params.monochrome_logs ? '' : "\033[0;30m";
    c_green = params.monochrome_logs ? '' : "\033[0;32m";
    c_yellow = params.monochrome_logs ? '' : "\033[0;33m";
    c_blue = params.monochrome_logs ? '' : "\033[0;34m";
    c_purple = params.monochrome_logs ? '' : "\033[0;35m";
    c_cyan = params.monochrome_logs ? '' : "\033[0;36m";
    c_white = params.monochrome_logs ? '' : "\033[0;37m";

    return """    ${c_dim}----------------------------------------------------${c_reset}
                                            ${c_green},--.${c_black}/${c_green},-.${c_reset}
    ${c_blue}        ___     __   __   __   ___     ${c_green}/,-._.--~\'${c_reset}
    ${c_blue}  |\\ | |__  __ /  ` /  \\ |__) |__         ${c_yellow}}  {${c_reset}
    ${c_blue}  | \\| |       \\__, \\__/ |  \\ |___     ${c_green}\\`-._,-`-,${c_reset}
                                            ${c_green}`._,._,\'${c_reset}
    ${c_dim}----------------------------------------------------${c_reset}
    """.stripIndent()
}

helpMessage()
exit 0
```

If run with `nextflow run script.nf` it only prints `foo` to console, whereas when I run it with
`NXF_ANSI_LOG=false nextflow run script.nf` it prints both nfcoreHeader() and `foo`to console.

**Comment by pditommaso** [comment]
Ohh, nf-core in technicolor .. 😂(let me try)

**Comment by pditommaso** [comment]
Wait, the `exit 0` is suspicious and surely it should only be used to report errors, not to termite successful executions. Can you try to remove it?

**Comment by phue** [comment]
Agreed, don't know why I put it there. Doesn't change the behaviour though.

**Comment by drpatelh** [comment]
@pditommaso Im seeing the same behaviour with another pipeline too. When `-ansi` is switched off the header is printed.

**Comment by pditommaso** [comment]
Ouch, now I'm able to replicate it.

**Comment by pditommaso** [comment]
Squashed. The nf-core banner now is printed tho there's a small glitch, but I think it's related to the ansi codes you are using 

```
[2m----------------------------------------------------
                                        ,--./,-.
        ___     __   __   __   ___     /,-._.--~'
  |\ | |__  __ /  ` /  \ |__) |__         }  {
  | \| |       \__, \__/ |  \ |___     \`-._,-`-,
                                        `._,._,'
----------------------------------------------------
```

**Comment by phue** [comment]
Great!

**Comment by pditommaso** [comment]
Included in version `19.04.1`.
